### ISSUE TITLE
Remove the image tag 'v' char in release page!

### ISSUE THREAD

**Issue by liuxuzxx** [grafana/loki#9422]
You may submit questions in any of the following ways:
  - On our [community forum](https://community.grafana.com/c/grafana-loki/41)
  - To our [mailing list](https://groups.google.com/forum/#!forum/lokiproject)
  - In `#loki` in [Grafana's Public Slack](https://slack.grafana.com/)

**NOTE: All questions submitted as issues will be closed.**

The docker pull command is error:
```bash
$ docker pull "grafana/loki:v2.8.2"
$ docker pull "grafana/promtail:v2.8.2"
```

the right docker pull command is:
```bash
$ docker pull "grafana/loki:2.8.2"
$ docker pull "grafana/promtail:2.8.2"
```
The loki and promtail image tag is 2.8.2 not is v2.8.2 ,So remove the 'v' char!

**Comment by liuxuzxx** [comment]
The release page: https://github.com/grafana/loki/releases

In version 2.8.0 2.8.1 2.8.2 the image tag must remove 'v' char!

**Comment by JStickler** [comment]
Duplicate of   [Issue #9384](https://github.com/grafana/loki/issues/9384)
Fixed by [PR #9425](https://github.com/grafana/loki/pull/9425)
